src/components/visualizations/common/utils/drilldownEventing.js

```diff
@@ -0,0 +1,146 @@
+// (C) 2007-2018 GoodData Corporation
+import { get, debounce } from 'lodash';
```

Comment:
I think we should use here `require` as import. As I remember right with this usage whole `lodash` lib is included in package. @vojty am I right ?

Reply:
No, there is babel package for this that extracts what i needed from lodash..